### ISSUE TITLE
La Pausa server space

### ISSUE THREAD

**Issue by daithi-coombes** [coder-forge/website#20]
PR: #21 
 The requirements:

 1. Somewhere to run development apps
 2. Somewhere for public collaboration

My current suggestions:
* [ ] Docker Ubuntu server - La Pausa installs, ftp, php etc etc etc - what evers needed
* [ ] Github la pausa repo
* [ ] La Pausa control over Meetup page (pay for professional account in meetup.com)

**Comment by alandodd888** [comment]
Hi Daithi, just to clarify we are answering the Slack stuff here tonight...thanks.

We're going for coderforge.io/lapausa rather than lapausa.coderforge.io due to SSL costs I presume?

**Comment by coder-forge-admin** [comment]
Yep that's it. Sorry couldn't make it in, got deadline for meeting
tomorrow.

Will get more done on ssl integration tonight. But 1st will get you that
ftp file setup
